playbooks/roles/lets-encrypt/tasks/main.yml

```diff
@@ -6,6 +6,10 @@
     state: directory
     recurse: yes
 
+- name: Find the current Let's Encrypt subscriber agreement URL
+  script: "../scripts/termsOfServiceLookup.py {{ le_api_endpoint }}"
+  register: le_agreement
+
```

Comment:
A "changed_when: False" could be used to avoid "changed" status every time the task is run.